Extension/src/LanguageServer/client.ts

```diff
@@ -978,26 +1019,29 @@ export class DefaultClient implements Client {
                 }
             }
         }
-        paths.push(localize("selectAnotherCompiler.string", "Select another compiler on my machine"));
+        paths.push(localize("selectAnotherCompiler.string", "Select another compiler on my machine..."));
         paths.push(localize("installCompiler.string", "Help me install a compiler"));
-        paths.push(localize("noConfig.string", "Do not configure a compiler (not recommended)"));
-        const index: number = await this.showSelectDefaultCompiler(paths);
+        paths.push(localize("noConfig.string", "Do not configure with a compiler (not recommended)"));
+        const index: number = await this.showSelectIntelliSenseConfiguration(paths, compilersOnly);
         let action: string = "";
+        let configurationSelected: boolean = false;
+        const statusBarIndicatorEnabled: boolean = await telemetry.showStatusBarIntelliSenseIndicator();
```

Comment:
For clarity, I would think "button" was a better description of what we're using over "indicator"

Reply:
Sure, that's just what it was named previously.

Reply:
changed

Reply:
> Sure, that's just what it was named previously.

Yeah, I know. It doesn't mean we have to live with it forever if there's a better option.